tasks/cli_file.yml

```diff
@@ -1,12 +1,17 @@
 - name: upload CLI file
+  become: true
```

Comment:
Perhaps all the 'become' statements in this role should use the 'jbosseap_become' role variable mentioned in issue #2.

Reply:
You just had to go and wait until you created that issue to comment on this, didn't you? :P